apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -54,7 +54,10 @@ const PlanUpdateSidePanel = () => {
 
   const snap = useOrgSettingsPageStateSnapshot()
   const visible = snap.panelKey === 'subscriptionPlan'
-  const onClose = () => snap.setPanelKey(undefined)
+  const onClose = () => {
+    router.push(router.asPath.split('?')[0], undefined, { shallow: true })
```

Comment:
Is this just to remove the query string from the URL? If so, I think there might be a better way of doing this

Reply:
^ yeah it is - what do you have in mind?

Reply:
Maybe something like:
```js
const router = useRouter()
const { x, ...queryWithoutX } = router.query
router.push(
  {
    pathname: router.pathname,
    query: queryWithoutX,
  },
  undefined,
  { shallow: true }
)
```

Reply:
^ ahhh makes sense, will give it a go

Reply:
okay! updated :) and also updated for the panels in project add ons too